menuconfig.py

```diff
@@ -320,7 +475,7 @@ def _expr_str(expr):
 def _main():
     menuconfig(standard_kconfig())
 
-def menuconfig(kconf):
+def menuconfig(kconf, style=None):
```

Comment:
Do you need the extra `style` argument here?

I think I'd prefer if `MENUCONFIG_STYLE` was the only mechanism for setting the style (it could still be set programmatically via `os.environ`). If it turns out to be limiting in some way later, we could add the argument.

Some earlier experience has taught me that once you add something, you're stuck with it forever, even if it wasn't really needed. Best to add just the minimal interface at first. :)